dbt_/models/intermediate/sub_indus/_int__sub_indus_models.yaml

```
@@ -0,0 +1,159 @@
+version: 2
+
+models:
+  - name: int__resultats_sub_indus_commune_annuel
+    description: Résultats CVM par commune pour la temporalité 'bilan_annuel'
+    tests:
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - inseecommune
+            - annee
+    columns:
+      - name: inseecommune
+        description: "Code INSEE de la commune"
+        type: VARCHAR
+        tests:
+          - not_null
+      - name: annee
+        description: "Annee du prevelvement"
+        type: SMALLINT
+      - name: categorie
+        description: "Catégorie du paramètre (pesticides, pfas, vcm, nitrates, etc.) selon la classification établie."
+        tests:
+          - accepted_values:
+              values: ["sub_indus_14DAN","sub_indus_PCLAT"]
+      - name: periode
+        description: "Période du résultat"
+        type: VARCHAR
+        tests:
+          - dbt_utils.expression_is_true:
+              expression: "LIKE 'bilan_annuel%'"
+      - name: nb_depassements
+      - name: nb_prelevements
+      - name: ratio_depassements
+        description: "Pourcentage de prélèvements qui dépassent la limite"
```

Ajouter ce détail important (il n'y a pas de limite reglementaire selon le brief) :

```suggestion
        description: "Pourcentage de prélèvements qui dépassent la limite sanitaire"
```